BlackPearl.Controls.Library/Control/MultiSelectCombobox.xaml.cs

```diff
@@ -279,6 +284,7 @@ private void Tb_Unloaded(object sender, RoutedEventArgs e)
 
             tb.Unloaded -= Tb_Unloaded;
             SelectedItems?.Remove(tb.Tag);
+            SelectionChanged?.Invoke(this, new SelectionChangedEventArgs(System.Windows.Controls.Primitives.Selector.SelectionChangedEvent, new [] { tb.Tag }, new List<object>()));
```

Comment:
Can we create a private function which will make code more readable and consistent?
```
void RaiseSelectionChangedEvent(IList removedItems, IList addedItems)
{
}
```
Also, lets stick to either array or list while passing parameter to this function. I see both are used interchangeably.